### DISCUSSION TITLE
running test suite files in predefined order

### DISCUSSION THREAD

**Discussion by skoduru-lucid** [avajs/ava#2753] [General]
Is there a way to run the test suite files in the pre defined order. I have 3 test suite files say file-A, file-B and file-C and i would like to run in order file-B->file-C->file-A

**Comment by novemberborn** [comment]
No. While we don't (yet) randomize the order, it is arbitrary, based on the underlying glob libraries.

The idea is that tests (and test files) should be independent of each other.
